Makefile

```diff
@@ -300,8 +308,27 @@ test:
 		TEST_TOKEN_UNALLOWED="$(shell cat $(BUILD_FOLDER)/auth-tokenUnallowedString)" \
 		ENABLE_AUTH="true" \
 		WITH_AMS_SUBSCRIPTIONS="true" \
+		ENABLE_KUBE_API="false" \
 		go test -v ./subsystem/... -count=1 $(GINKGO_FOCUS_FLAG) -ginkgo.v -timeout 120m
 
+enable-kube-api-for-subsystem: $(BUILD_FOLDER)
+	skipper make deploy-service-requirements ENABLE_KUBE_API=true ENABLE_AUTH=false
```

Comment:
As far as i know, In jenkins skipper doesn't container permissions to minikube, so if you run minikube commands from skipper they will fail. 
What you need to do is to install the requirements for this command on the host, @oshercc can you help @RazRegev to install 
```
RUN curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | \
    bash -s -- 3.8.8 && mv kustomize /usr/bin/
RUN go get -u  sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.0 
```
I coped it from the build container

Reply:
@oshercc who need to do it? is it some thing that @RazRegev or you need to do?